### DISCUSSION TITLE
Ventoy is awesome, however its UI needs to be more intuitive.

### DISCUSSION THREAD

**Discussion by malnaanah** [ventoy/Ventoy#1807] [Ideas]
Ventoy is awesome, and I recommend it for everybody I know. However the UI of ventoy is terrible, especially on Linux, and dealing with it is hard for new users, I believe the following points should be be taken into account to make its design more intuitive.

1. Provide an installer to create menu launcher.
2. Provide a single UI for installation and configuration with intuitive and unified design.
3. Automatic detection for USB stick, especially on Linux, to avoid mistaking hard drive with USB stick in command line.
4. Automatic detection for architecture, i386, x86_64, etc.
5. Automatic creation for ISO folder to search within, and making it as a default choice since it is needed by most users, especially if many files are copied to the USB stick.
6. Make Linux_remount option active by default, since its needed by most users.

I hope my comments will be taken into consideration for the best of Ventoy.

Best regards.

**Comment by ventoy** [comment]
Thanks for your comments.

1. This will need `.rpm/.deb/...` package and may be provided in the future but not in the near future.
2. Did you mean configuration UI is the VentoyPlugson Web page? If so, VentoyPlugson is for advanced options and most userd don't need it.
3. In the GUI (GTK/QT/WebUI) Ventoy does auto detect for USB stick.
4. If `1` is implemented
5. I prefer to give user a fully empty partition.
6. This is still an experimental option and not fully tesed.
